python/tests/test_variable.py

```diff
@@ -321,6 +321,15 @@ def test_2D_access_variances():
     assert np.array_equal(var.variances, np.ones(shape=(2, 3)))
 
 
+def test_values_lifetime():
```

Comment:
We have a special file for lifetime tests, `test_lifetime.py`, consider putting this there.